src/app/containers/Figure/__snapshots__/index.test.jsx.snap

```diff
@@ -547,18 +547,18 @@ exports[`Figure should render an image with caption text with inline link 1`] =
 }
 
 .c3 {
-  font-size: 0.9375rem;
+  font-size: 0.9375em;
   line-height: 1.125rem;
   background-color: #D5D0CD;
   color: #404040;
-  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-family: ReithSansNewsRegular,Helvetica,Arial,sans-serif;
   padding: 0.5rem;
   width: 100%;
 }
 
 @media (min-width:37.5em) {
   .c3 {
-    font-size: 0.875rem;
+    font-size: 0.875em;
```

Comment:
With this (and others) changing from `rem` to `em` this seems like a regression in the package version of `gel-foundations` and this [PR](https://github.com/bbc/psammead/pull/290/files). Would be worth double checking which version of `gel-foundations` you have installed locally